modules/andes-core/client/src/main/java/org/wso2/andes/jms/JMSContext.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.andes.jms;
+
+/**
+ * andes extension of JMSContext.
+ */
+public interface JMSContext extends javax.jms.JMSContext {
```

Comment:
Do we plan to have any extended operations specific to Andes in JMSContext ? Otherwise this is redundant ?